### PR TITLE
Config option to override initial ship spawns. Now includes combat ships.

### PR DESCRIPTION
**Pull request by Jackriip**
…ip pairs

## About The Pull Request

This adds config options to change the initial ship spawning behavior. You can add more neutral ships. Additionally you can allocate a pair of combat ships. The combat ships are not balanced against each other (You can get a small NT and a large Syn). 

I set neutral ships to spawn 1 and the combat pair to 1.

## Why It's Good For The Game

Adds more non-passwordable ships round start. Adds combat ships to the mix which should alleviate some of the lack of combat concerns.

## Changelog
:cl:
add: config for additional round start neu ships
add: config for round start combat ships
/:cl:

### CODE REVIEW

**Comment by Jackriip** [comment]
test merging

File: voidcrew/code/controllers/subsystem/overmap.dm [hunk 1]
```diff
@@ -168,19 +174,38 @@ SUBSYSTEM_DEF(overmap)
...
 
-/datum/controller/subsystem/overmap/proc/spawn_initial_ships()
+/datum/controller/subsystem/overmap/proc/spawn_initial_ships(num)
 #ifndef UNIT_TESTS
-	var/datum/map_template/shuttle/selected_template = SSmapping.maplist[pick(SSmapping.maplist)]
-	INIT_ANNOUNCE("Loading [selected_template.name]...")
-	SSshuttle.load_template(selected_template)
+	var/templates = list()
+	log_admin("number of ships loaded: [num]")
+	for(var/i=1,i<=num,i++)
```
Comment (SuperNovaa41) [inline voidcrew/code/controllers/subsystem/overmap.dm]:
```suggestion
	for(var/i in 1 to num)
```
byond style for loops are a little more performant

File: voidcrew/code/controllers/subsystem/overmap.dm [hunk 2]
```diff
@@ -168,19 +174,38 @@ SUBSYSTEM_DEF(overmap)
...
 		query_round_map_name.Execute()
 		qdel(query_round_map_name)
 #endif
 
+/**
+  * Creates a pair of combat shuttles
+  */
+/datum/controller/subsystem/overmap/proc/spawn_initial_combat_ships(num)
+#ifndef UNIT_TESTS
+	for(var/i=1,i<=num,i++)
```
Comment (SuperNovaa41) [inline voidcrew/code/controllers/subsystem/overmap.dm]:
```suggestion
	for(var/i in 1 to num)
```

File: voidcrew/code/controllers/subsystem/overmap.dm [hunk 3]
```diff
@@ -168,19 +174,38 @@ SUBSYSTEM_DEF(overmap)
...
 			spawn_event_cluster(type, T, chance / 2)
 
-/datum/controller/subsystem/overmap/proc/spawn_initial_ships()
+/datum/controller/subsystem/overmap/proc/spawn_initial_ships(num)
 #ifndef UNIT_TESTS
-	var/datum/map_template/shuttle/selected_template = SSmapping.maplist[pick(SSmapping.maplist)]
-	INIT_ANNOUNCE("Loading [selected_template.name]...")
-	SSshuttle.load_template(selected_template)
+	var/templates = list()
+	log_admin("number of ships loaded: [num]")
```
Comment (SuperNovaa41) [inline voidcrew/code/controllers/subsystem/overmap.dm]:
```suggestion
	log_admin("\[SHUTTLE]: Number of ships loaded: [num]")
```

File: voidcrew/code/controllers/subsystem/overmap.dm [hunk 4]
```diff
@@ -168,19 +174,38 @@ SUBSYSTEM_DEF(overmap)
...
 
+/**
+  * Creates a pair of combat shuttles
+  */
+/datum/controller/subsystem/overmap/proc/spawn_initial_combat_ships(num)
+#ifndef UNIT_TESTS
+	for(var/i=1,i<=num,i++)
+		var/datum/map_template/shuttle/nt_ship = SSmapping.nt_ship_list[pick(SSmapping.nt_ship_list)]
+		var/datum/map_template/shuttle/syn_ship = SSmapping.syn_ship_list[pick(SSmapping.syn_ship_list)]
+		INIT_ANNOUNCE("Loading [nt_ship.name]...")
```
Comment (SuperNovaa41) [inline voidcrew/code/controllers/subsystem/overmap.dm]:
```suggestion
		INIT_ANNOUNCE("'\[SHUTTLE]: Loading [nt_ship.name]...")
```

File: voidcrew/code/controllers/subsystem/overmap.dm [hunk 5]
```diff
@@ -168,19 +174,38 @@ SUBSYSTEM_DEF(overmap)
...
+		SSshuttle.load_template(nt_ship)
+		INIT_ANNOUNCE("Loading [syn_ship.name]...")
```
Comment (SuperNovaa41) [inline voidcrew/code/controllers/subsystem/overmap.dm]:
```suggestion
		INIT_ANNOUNCE("\[SHUTTLE]: Loading [syn_ship.name]...")
```

**Review by SuperNovaa41 — CHANGES_REQUESTED** [review]

**Pull request merged**